src/Grid/index.jsx

```diff
@@ -51,85 +66,92 @@ const Grid = forwardRef( ( props, ref ) =>
 Grid.propTypes =
 {
     /**
-     * Vertical alignment of the grid items
+     * Block-axis (usually vertical) alignment of the grid content
      */
-    align         : PropTypes.oneOf( [ 'start', 'center', 'end', 'stretch' ] ),
+    alignContent : PropTypes.oneOf( [ 'start', 'center', 'end', 'stretch' ] ),
     /**
-     * Defines the size of implicitly set columns
+     * Block-axis (usually vertical) alignment of the grid items
      */
-    autoCols      : PropTypes.string,
+    alignItems   : PropTypes.oneOf( [ 'start', 'center', 'end', 'stretch' ] ),
+
     /**
-     * Controls where to auto place new grid items if their place is
-     * undefined
+     * Defines the size of implicit grid columns
      */
-    autoFlow      : PropTypes.oneOf( [ 'row', 'col' ] ),
+    autoColumns : PropTypes.string,
     /**
-     * Defines the size of implicitly set rows
+     * Controls placement of items outside the explicit grid
      */
-    autoRows      : PropTypes.string,
+    autoFlow    : PropTypes.oneOf( [ 'row', 'column' ] ),
     /**
-     *  Grid content (Columns)
+     * Defines the size of implicit grid rows
      */
-    children      : PropTypes.node,
+    autoRows    : PropTypes.string,
     /**
-     *  CSS class name
+     *  Grid content
      */
-    className     : PropTypes.string,
+    children    : PropTypes.node,
     /**
-     *  Column gap
+     *  CSS class name
      */
-    columnGap     : PropTypes.oneOf( [ 'none', 'S', 'M', 'L' ] ),
+    className   : PropTypes.string,
     /**
-     *  Number of columns - should be an integer > 0
+     *  Number of columns (integer > 0) or array of column sizes (CSS length
+     *  strings)
      */
-    columns       : PropTypes.number,
+    columns     : PropTypes.oneOfType(
+        PropTypes.number,
+        PropTypes.arrayOf( PropTypes.string ),
+    ),
     /**
      *  CSS class map
      */
-    cssMap        : PropTypes.objectOf( PropTypes.string ),
+    cssMap : PropTypes.objectOf( PropTypes.string ),
     /**
-     *  Custom sizes of columns
+     *  Gap between rows and/or columns
      */
-    customColumns : PropTypes.string,
+    gap    : PropTypes.oneOfType(
+        PropTypes.oneOf( [ 'none', 's', 'm', 'l' ] ),
+        PropTypes.arrayOf( [ PropTypes.oneOf( [ 'none', 's', 'm', 'l' ] ) ] ),
```

Comment:
Changed these to lowercase to match what we export in theme vars... problem is this is inconsistent with what we have else where (we’ve generally used uppercase). I guess we should either:
- update the theme vars export to be uppercase, or
- update all component props to use lowercase

... thoughts?

Reply:
I think latter is better: update all component props to use lowercase - most components already have lowercase prop values whenever prop value is a string or an array of strings.